metabase/metabase
# Snowplow: Add new event for search results filtered

Closes https://github.com/metabase/metabase/issues/35122

### Description

Adds snowplow schema, events, and e2e tests for tracking when search filters are used within the search app.

### How to verify

1. Ensure that Snowplow is set up according to the [Snowplow integration] page (https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa?pvs=4)
2. Go to the search app, and type in a search
3. Apply any filter
4. Access Snowplow, and see that there are no bad events, and that there is a new `search_results_filtered` event

### Checklist
